keyboards/laptreus/config.h

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2019 Chuck "@vosechu" Lauer Vose <vosechu@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+
+#define VENDOR_ID       0x1209 // pid.codes Vendor ID
+#define PRODUCT_ID      0xA1F1 // FIXME: Need to send PR to pid.codes to make this official
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Vosechu
+#define PRODUCT         Laptreus
+#define DESCRIPTION     keyboard
```

Comment:
This should probably be more descriptive. :)

It's not necessary, but ....

Reply:
I totally agree! Let me try something out. When I had it longer the first time it made the USB menu obscenely long; like the whole description was there every time I clicked on the device! But that seems crazy and I'd like to try it again now that I've learned a bit more and gotten more comfortable with the board.

Reply:
`DESCRIPTION` is only used in two places: the Command version info printout and the Adafruit BLE device name:

https://github.com/qmk/qmk_firmware/blob/d8e9a0f7a319e27c8dbb4e5a1131bc02b365da76/tmk_core/protocol/lufa/adafruit_ble.cpp#L536-L537

IMO it probably shouldn't be there.

Reply:
![screen shot 2019-02-01 at 4 59 54 pm](https://user-images.githubusercontent.com/11463/52157299-de40da80-2642-11e9-82cc-e517309649d3.png)

Finally tested this out: I'm not sure how it gets there, but this is what's in my bluetooth bar. :/ 

@fauxpark Are you saying this is a bug in adafruit_ble.cpp? Or is there something I can do to fix this?

Reply:
Nevermind! I see the ticket you created. Thank you @fauxpark!